res/commands.json

```diff
@@ -340,7 +340,7 @@
 	"FUN_ROLL": {
 		"brief": "Gets a random number.",
 		"usage": "roll <min> <max>\nroll <max>",
-		"additional": "**Example:\n** `!roll 4 10`: Gets a random number between `4` and `10` (inclusive).\n `roll 10`: Gets a random number from `0` to `10` (inclusive).",
+		"additional": "**Example:\n** `!roll 4 10`: Gets a random number between `4` and `10` (inclusive).\n `!roll 10`: Gets a random number from `0` to `10` (inclusive).",
```

Comment:
`!` is not necessarily the prefix being used!

Reply:
It is in the commands.json. I have followed the `!top` command 
![image](https://user-images.githubusercontent.com/69952230/100264179-b38ee700-2f78-11eb-8052-a94758acc237.png)

Reply:
ah crap. all right then